src/main/resources/org/jenkins/plugins/lockableresources/LockStep/help-resource.html

```diff
@@ -0,0 +1,5 @@
+<div>
+	<p>
+		The resource name to lock as defined in Global settings.
```

Comment:
Hmm, I guess this means you cannot implicitly define a resource just by starting to lock it? Goes against the grain of Pipeline-as-code.